aten/src/ATen/native/AdaptiveMaxPooling2d.cpp

```diff
@@ -11,6 +11,53 @@
 #include <ATen/ops/adaptive_max_pool2d_native.h>
 #endif
 
+/**
```

Comment:
I think it is ok to remove this comment and just have the updates in the user-facing documentation